core/boot.js

```diff
@@ -85,6 +85,13 @@ async function initCore({ghostServer, config}) {
     await settings.syncEmailSettings(config.get('hostSettings:emailVerification:verified'));
     debug('End: settings');
 
+    // Custom theme settings cache depends on it's backing model.
+    // Doesn't do anything except create a class instance but needs to be ready before it's required in frontend
+    debug('Begin: Custom theme settings');
+    const customThemeSettingsCache = require('./shared/custom-theme-settings-cache');
+    customThemeSettingsCache.init({model: models.CustomThemeSetting});
+    debug('End: Custom theme settings');
+
```

Comment:
I think that can be avoided if the cache population is done from the custom theme settings service rather than in the cache instance itself.